app.js

```diff
@@ -1,5 +1,122 @@
+    const player = checkedRadio.value;
+    //console
+    //console.log(player);
+
+    //create a randomizer for the computer
+    const getRandomThrow = Math.floor(Math.random() * 3);
+    //console.log(getRandomThrow);
+
+    //create function that converts computerPlay into computerThrow
+    function computerThrow() {
+        if (getRandomThrow === 0) {
+            return 'rock';
+        }
+        if (getRandomThrow === 1) {
+            return 'paper';
+        }
+        if (getRandomThrow === 2) {
+            return 'scissors';
+        }
+    }
+    const computer = computerThrow();
+
+    //console.log(computer);
+    //console.log(player);
+
+    //add function with nine different win loss draw scenarios
+
+    //rock scenarios
+    if (player === "rock" && computer === "rock") {
+        drawData++;
+        winLoseDraw.textContent = 'Rock ties Rock, its a draw!';
+        //console.log('hello');
+    }
+    if (player === "rock" && computer === "paper") {
+        lossData++;
+        winLoseDraw.textContent = 'Paper beats Rock, you lose!';
+        //console.log('hello');
+    }
+    if (player === "rock" && computer === "scissors") {
+        winData++;
+        winLoseDraw.textContent = 'Rock beats Scissors, you win!';
+        //console.log('hello');
+    }
+
+    //paper scenarios
+    if (player === "paper" && computer === "rock") {
+        winData++;
+        winLoseDraw.textContent = 'Paper beats Rock, you win!';
+        //console.log('hello');
+    }
+    if (player === "paper" && computer === "paper") {
+        drawData++;
+        winLoseDraw.textContent = 'Paper ties Paper, its a draw!';
+        //console.log('hello');
+    }
+    if (player === "paper" && computer === "scissors") {
+        lossData++;
+        winLoseDraw.textContent = 'Scissors beats Paper, you lose!';
+        //console.log('hello');
+    }
+
+    //scissors scenarios
+    if (player === "scissors" && computer === "scissors") {
+        drawData++;
+        winLoseDraw.textContent = 'Scissors ties Scissors, its a draw!';
+        //console.log('hello');
+    }
+    if (player === "scissors" && computer === "paper") {
+        winData++;
+        winLoseDraw.textContent = 'Scissors beats Paper, you lose!';
+        //console.log('hello');
+    }
+    if (player === "scissors" && computer === "rock") {
+        lossData++;
+        winLoseDraw.textContent = 'Rock beats Paper, you lose!';
+        //console.log('hello');
+    }
+    
+    //update spans to reflect new score
+    wins.textContent = `Wins: ${winData}`;
+    losses.textContent = `Losses: ${lossData}`;
+    draws.textContent = `Draws: ${drawData}`;
+
+
+
+    //console wins draws and losses
+    //console.log(winData);
+    //console.log(lossData);
+    //console.log(drawData);
+
 
-// set event listeners to update state and DOM
\ No newline at end of file
+}
+);
+//add reset button
+reset.addEventListener('click', () => {
+    window.location.reload();
```

Comment:
This works, but think more about how to reset state rather than reload the page. So, reset functionality should be something like: 

```js
winData = 0;
lossData = 0;
drawsData = 0;

wins.textContent = `Wins: ${winData}`;
losses.textContent = `Losses: ${lossData}`;
draws.textContent = `Draws: ${drawData}`;
```